src/RestGroup.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Yiisoft\Yii\Rest;
+
+use Yiisoft\Http\Method;
+use Yiisoft\Router\Group;
+use Yiisoft\Router\Route;
+use Yiisoft\Router\RouteCollectorInterface;
+
+class RestGroup
+{
+    public static function create(string $prefix, string $controller): RouteCollectorInterface
+    {
+        $routes = self::createDefaultRoutes($prefix, $controller);
+        $group = Group::create($prefix, $routes);
+
+        // add middlewares here
```

Comment:
What middlewares?

Reply:
I want to make middleware that will convert any variables to the `ResponseInterface`